code/ui/components/src/components/form/input/input.tsx

```diff
@@ -207,7 +207,7 @@ type TextareaProps = Omit<
   valid?: ValidationStates;
   height?: number;
 } & React.RefAttributes<HTMLTextAreaElement>;
-export const Textarea: FC<TextareaProps> = Object.assign(
+export const Textarea = Object.assign(
   styled(
     forwardRef<any, TextareaProps>(function Textarea({ size, valid, align, ...props }, ref) {
```

Comment:
Is `children` included somewhere in `TextareaProps`? I didn't see it.

Reply:
No but it shouldn't need it anyway for an input Element, right?

Reply:
Oh, the name threw me. I thought it was rendering a `<textarea>`.

Reply:
Indeed, it is:

```ts
type TextareaPropsRaw = React.TextareaHTMLAttributes<HTMLTextAreaElement>;
```

Reply:
@kylegach I don't think passing `children` to this `Textarea` is valid anyway, so it seems like this fixes a type bug, removing that possibility?